README.md

```diff
@@ -13,6 +13,12 @@ and away you go with a full-stack blockchain and command line tool for transacti
 
 WARNING: Currently uses plain-text private keys for transactions and is otherwise not production ready.
 
+## Prerequisites
+
+* Go to https://golang.org/doc/install to install Golang. 
+* Once you have Golang installed you will also need to install the Golang package manager `glide`: https://github.com/Masterminds/glide. 
```

Comment:
in that case I think we need to change the instructions here: 

>>> 
The guaranteed correct way of compiling from source, assuming you've already run go get or otherwise cloned the repo, is:

```cd $GOPATH/src/github.com/tendermint/basecoin
git checkout develop # (until we release tendermint v0.9)
make get_vendor_deps
make install
```
because it seems like once you do `go get` you're supposed to do all that stuff ^^. How would you change it?